Runtime/Scripts/RGBotConfigs/RGEntity.cs

```diff
@@ -38,8 +38,23 @@ void Start()
 
             if (actionMap.Count > 0)
             {
-                RGDebug.LogDebug($"Entity registered with {actionMap.Count} actions");
+
+                // If this is an entity with other actions, also give it default actions
+                var defaultActions = new RGAction[]
+                {
+                    gameObject.AddComponent<DrawLineToAction>(),
```

Comment:
instead of putting these debug actions on every entity, i would restructure them as a special case where the from entity and to entity/position/etc are part of the payload.

Adding these to all those game objects seems weird when it is debug code.

Let's separate debug from actual code and make debug be debug.

Reply:
The gameObject with the BotServerListener/etc on it can have a debugactionhandler on it that handles rendering of all debug actions.